src/ballet/sbpf/fd_sbpf_loader.c

```diff
@@ -546,6 +545,12 @@ fd_sbpf_program_new( void *                     prog_mem,
     return NULL;
   }
 
+  /* https://github.com/solana-labs/rbpf/blob/v0.8.0/src/elf_parser/mod.rs#L99 */
+  if( FD_UNLIKELY( !fd_ulong_is_aligned( (ulong) rodata, 8UL ) ) ){
+    FD_LOG_WARNING(( "rodata is not 8-byte aligned" ));
+    return NULL;
+  } 
```

Comment:
In some places in the BPF loaders we allocate rodata with 32-byte alignment, should we change these to be 8-byte aligned?

Reply:
nice catch, free (best case) 24 bytes!!